collect_app/src/main/java/org/odk/collect/android/utilities/PRNGFixes.java

```diff
@@ -39,107 +36,13 @@
  */
 public final class PRNGFixes {
```

Comment:
Isn't this whole class unused now?

Reply:
Yes, actually @grzesiek2010! :sweat_smile: